The gvm fails to build and the --no-allow-shlib-undefined flag is unknown.
Index: tremfusion-0.99~r3/Makefile
===================================================================
--- tremfusion-0.99~r3/Makefile	2009-07-05 22:07:05.000000000 +0200
+++ tremfusion-0.99~r3/Makefile	2012-08-01 19:57:45.000000000 +0200
@@ -34,7 +34,7 @@
   BUILD_GAME_SO    = 1
 endif
 ifndef BUILD_GAME_QVM
-  BUILD_GAME_QVM   = 1
+  BUILD_GAME_QVM   = 0
 endif
 
 #############################################################################
@@ -398,7 +398,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS) 
 
   BASE_CFLAGS+=-I/usr/X11R6/include
   CLIENT_LDFLAGS=-L/usr/X11R6/$(LIB)
@@ -595,7 +595,7 @@
 
   SHLIBEXT=dylib
   SHLIBCFLAGS=-fPIC -fno-common
-  SHLIBLDFLAGS=-dynamiclib $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-dynamiclib $(LDFLAGS) 
 
   NOTSHLIBCFLAGS=-mdynamic-no-pic
 
@@ -672,7 +672,7 @@
 
   SHLIBEXT=dll
   SHLIBCFLAGS=
-  SHLIBLDFLAGS=-shared $(LDFLAGS)--no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS)
 
   BINEXT=.exe
 
@@ -809,7 +809,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS) 
 
   # don't need -ldl (FreeBSD)
   LIBS+=-lm
@@ -868,7 +868,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS) 
 
   LIBS=-lm
 
@@ -895,7 +895,7 @@
   LIBS=-lm
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS) 
   CLIENT_LIBS=-lpthread
 
   BASE_CFLAGS = -Wall -fno-strict-aliasing -Wimplicit -Wstrict-prototypes
@@ -929,7 +929,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=
-  SHLIBLDFLAGS=-shared --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared 
 
   LIBS=-ldl -lm -lgen
   # FIXME: The X libraries probably aren't necessary?
@@ -994,7 +994,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared $(LDFLAGS) --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared $(LDFLAGS) 
 
   LIBS=-lsocket -lnsl -ldl -lm
 
@@ -1011,7 +1011,7 @@
 
   SHLIBEXT=so
   SHLIBCFLAGS=-fPIC
-  SHLIBLDFLAGS=-shared --no-allow-shlib-undefined
+  SHLIBLDFLAGS=-shared 
 
 endif #Linux
 endif #darwin
